src/Constants/Options.ts

```diff
@@ -0,0 +1,226 @@
+import { AuthType, Grades, Option } from '@/types';
+
+export const TestTypeOptions: Option[] = [
+  { value: 'assessment', label: 'Assessment' },
+  { value: 'homework', label: 'Homework' },
+  { value: 'omr-assessment', label: 'Omr-Assessment' },
+];
+
+export const SessionTypeOptions: Option[] = [
+  { value: 'sign-in', label: 'SignIn' },
+  { value: 'sign-up', label: 'SignUp' },
+  { value: 'sign-in with forgot id', label: 'SignIn with forgot id' },
```

Comment:
"Forgot ID" capitalize

Reply:
Should I Capitalize in label only or the value also?